cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -226,23 +330,23 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
         """
           |One way of filtering the output from `SHOW INDEXES` by index type is the use of type keywords,
           |listed in the <<administration-indexes-syntax, syntax table>>.
-          |For example, to show only b-tree indexes, use `SHOW BTREE INDEXES`.
+          |For example, to show only range indexes, use `SHOW RANGE INDEXES`.
           |Another more flexible way of filtering the output is to use the `WHERE` clause.
           |An example is to only show indexes not belonging to constraints.""".stripMargin,
-      prepare = _ => executePreparationQueries(List("create index for ()-[r:KNOWS]-() on (r.since)")),
-      queryText = "SHOW BTREE INDEXES WHERE uniqueness = 'NONUNIQUE'",
+      prepare = _ => executePreparationQueries(List("create range index for ()-[r:KNOWS]-() on (r.since)")),
+      queryText = "SHOW RANGE INDEXES WHERE uniqueness = 'NONUNIQUE'",
       optionalResultExplanation =
         """This will only return the default output columns.
           |To get all columns, use `SHOW INDEXES YIELD * WHERE ...`.""".stripMargin,
-      assertions = p => assertEquals(4, p.size)
+      assertions = p => assertEquals(1, p.size)
     )
   }
 
   @Test def drop_index_on_a_label_single_property() {
     prepareAndTestQuery(
       title = "Drop a single-property index",
-      text = "An index on all nodes that have a label and single property combination can be dropped with `DROP INDEX ON :Label(property)`.",
-      prepare = _ => executePreparationQueries(List("create index for (p:Person) on (p.firstname)")),
+      text = "A b-tree index on all nodes that have a label and single property combination can be dropped with `DROP INDEX ON :Label(property)`.",
```

Comment:
As discussed with @emmaholmbergohlsson we should do this for constraints as well